docs/getting-started/installation.md

```diff
@@ -27,6 +27,37 @@ Chart.js built files are also available through [jsDelivr](https://www.jsdelivr.
 
 <https://www.jsdelivr.com/package/npm/chart.js?path=dist>
 
+## Security Considerations
+
+When loading Chart.js from a CDN, it's recommended to use **Subresource Integrity (SRI)** to protect against tampered or malicious scripts. Here are secure examples:
+
+### CDNJS with SRI
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/4.5.0/chart.min.js" 
+        integrity="sha512-n/G+dROKbKL3GVngGWmWfwK0yPctjZQM752diVYnXZtD/48agpUKLIn0xDQL9ydZ91x6BiOmTIFwWjjFi2kEFg==" 
+        crossorigin="anonymous" 
+        referrerpolicy="no-referrer">
+</script>
+```
+
+### jsDelivr with SRI
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/chart.js@4.5.0/dist/chart.min.js" 
+        integrity="sha512-n/G+dROKbKL3GVngGWmWfwK0yPctjZQM752diVYnXZtD/48agpUKLIn0xDQL9ydZ91x6BiOmTIFwWjjFi2kEFg==" 
+        crossorigin="anonymous" 
+        referrerpolicy="no-referrer">
+</script>
+```
+
+**Benefits of SRI:**
+- Prevents execution of tampered scripts from CDNs
+- Protects against supply chain attacks
+- Ensures script integrity in production environments
+
+You can generate integrity hashes for any Chart.js version using tools like [SRI Hash Generator](https://www.srihash.org/) or find them on [cdnjs.com](https://cdnjs.com/libraries/Chart.js).
```

Comment:
The part about using external tools should be removed and only the part of keeping the hash of the CDN should be used. 
We have no way of knowing if the linked tool is giving out false hashes.